src/vdom.rs

```diff
@@ -153,18 +170,42 @@ impl<Ms: 'static, Mdl: 'static, ElC: View<Ms>, GMs> ::std::fmt::Debug for App<Ms
     }
 }
 
+type InitAppBuilder<Ms, Mdl, ElC, GMs> = AppBuilder<
+    Ms,
+    Mdl,
+    ElC,
+    GMs,
+    (),
+    (),
+    (),
+    InitFn<Ms, Mdl, ElC, GMs>,
+    builder::MountPointInitInitAPI<(), InitFn<Ms, Mdl, ElC, GMs>>,
```

Comment:
Typo? (`MountPointInitInitAPI` vs `MountPointInitAPI`)

Reply:
It's the `(MountPoint[And]Init)InitAPI`, so not a typo. But the name is a bit weird. I was going to name it `LegacyInitAPI` but that didn't tell me what it did, so I went with this one instead. If you have a better idea, I'd be glad to hear it, though.

Reply:
I didn't dive into the logic, so I don't know the purpose. But you are saying it can be named `LegacyInitAPI`, so I assume it will be deleted / refactored in the future. If it's true, just mark it `deprecated` or add comment like `will be removed once the new API is implemented` and we don't have to waste time on it now.